pulseapi/profiles/serializers.py

```diff
@@ -96,6 +96,11 @@ class UserProfileSerializer(serializers.ModelSerializer):
         allow_blank=True,
     )
 
+    is_active = serializers.BooleanField(
+        required=False,
+        allow_blank=True,
```

Comment:
remove this since it's not a valid option for `BooleanField`

Reply:
yeah probably...I don't think it needs the `required` option since it will always be there.

Reply:
done